docs/source/docker/advanced/datapool.rst

```diff
@@ -67,6 +67,27 @@ first time.
     |-- passageway1-c1.avi
     `-- terrace1-c0.avi
 
+Let's create a Lightly dataset which uses that bucket (choose your tab - S3, GCS or Azure):
+
+.. tab-set::
+  .. tab-item:: AWS S3 Datasource
+
+    .. literalinclude:: ./code_examples/python_create_dataset_s3_example.py
+      :linenos:
+      :language: python
+
+  .. tab-item:: GCS Datasource
+
+    .. literalinclude:: ./code_examples/python_create_dataset_gcs_example.py
+      :linenos:
+      :language: python
+
+  .. tab-item:: Azure Datasource
+
+    .. literalinclude:: ./code_examples/python_create_dataset_azure_example.py
+      :linenos:
+      :language: python
+
 Now we can run the following code to select a subset based on the 
 :code:`'stopping_condition_minimum_distance': 0.1` stopping condition. In a first,
 selection run we only select images with the specific minimum distance between 
```

Comment:
Maybe add a note below this paragraph (or the code snippet) that we use `client.set_dataset_id_by_name` instead of `client.create_dataset` to allow running the snippet multiple times on the same dataset. And that `create_dataset` will throw an error if a dataset with this name already exists.

You could also highlight the `client.set_dataset_id_by_name` line with:
```
.. code-block:: python
    :emphasize-lines: 8
```

Reply:
done